javacord-api/src/main/java/org/javacord/api/entity/intent/IntentType.java

```diff
@@ -0,0 +1,182 @@
+package org.javacord.api.entity.intent;
+
+/**
+ * Represents an intent type.
+ *
+ * @see <a href="https://discordapp.com/developers/docs/topics/gateway#list-of-intents">Discord docs</a>
+ */
+public enum IntentType {
```

Comment:
All of these are `Intent`s and not `IntentType`s.
The getter and setters should also be adjusted accordingly (I suggested changes to some of them to give you an idea of the naming).

```suggestion
public enum Intent {
```